src/KafkaFlow/Consumers/Consumer.cs

```diff
@@ -34,9 +37,8 @@ internal class Consumer : IConsumer
         _dependencyResolver = dependencyResolver;
         _logHandler = logHandler;
         this.Configuration = configuration;
-        _flowManager = new ConsumerFlowManager(
-            this,
-            _logHandler);
+        _flowManager = new ConsumerFlowManager(this, _logHandler);
+        _maxPollIntervalExceeded = new(_logHandler);
```

Comment:
Is it possible to add tests to these scenarios?

Reply:
Without a well-defined integrated test strategy, I would say that the answer is no.